lbrynet/extras/daemon/Daemon.py

```diff
@@ -3258,179 +3258,108 @@ def jsonrpc_routing_table_get(self):
     Create and list comments.
     """
 
-    @requires(WALLET_COMPONENT)
-    async def jsonrpc_comment_list(self, claim_id, parent_comment_id=None, flat=False,
-                                   page=1, page_size=None, max_replies_shown=5):
+    async def jsonrpc_comment_list(self, claim_id: str, parent_id: str = None,
+                                   page=1, page_size=50, include_replies=True):
         """
         List comments associated with a claim.
 
         Usage:
-            comment_list <claim_id> [--flat] [(--page=<page> --page_size=<page_size>)]
-                         [--parent_comment_id=<parent_comment_id>]
-                         [--max_replies_shown=<max_replies_shown>]
+            comment_list    (<claim_id> | --claim_id=<claim_id>)
+                            [(--page=<page> --page_size=<page_size>)]
+                            [--parent_id=<parent_id>] [--include_replies]
 
         Options:
-            --flat  : (bool) Flag to indicate whether or not you want the
-                                    replies to be flattened along with the rest of
-                                    the comments attached to the claim. Off by default
-            --parent_comment_id=<parent_comment_id>  : (int) The ID of an existing
-                                                             comment to list replies from
-            --max_replies_shown=<max_replies_shown>  : (int) For every comment that we pull replies from,
-                                                             only retrieve up to this amount.
-                                                             Note: This is not the same as page size.
+            --include_replies  : (bool) Flag to indicate whether or not you want the
```

Comment:
shouldn't these options also include --claim_id and --comment? I think that's how other parts of the API are done.

Reply:
If `claim_id` and `comment` are mandatory then they aren't listed in `options`

Reply:
Options section doesn't mean it's optional. It's the section where you describe the meaning of arguments. http://docopt.org/

Reply:
fixed